README.md

```diff
@@ -105,7 +105,15 @@ node server --help
 
 Open your browser at the following address: [http://localhost:8280](http://localhost:8280)
 
-Enjoy!
+If you want to access `lncli-web` using a domain, add the `--serverhost` flag like so:
+
+ ```
+ node server --serverhost
```

Comment:
May be you should add your custom domaine placeholder:

Something like: `node server --serverhost <custom host name>`

Reply:
I assume adding a domain after `--serverhost` should limit access to that domain. Whether I add one or not, all domains on the server I'm using can access lncli-web when I add the `8280` port.

What's the desired outcome of adding a domain after the tag?

Reply:
Server host value is used here:

https://github.com/mably/lncli-web/blob/master/app/server.js#L95

Didn't know that an empty would be used as a wildcard.

I guess best practice is to limit access to particular hostname.